python/metatensor-torch/metatensor/torch/atomistic/io.py

```diff
@@ -0,0 +1,148 @@
+import io
+import os
+from pathlib import Path
+from typing import Union
+
+import numpy as np
+
+import torch
+from metatensor.torch import load_block_buffer
+from metatensor.torch import save_buffer as metatensor_torch_save_buffer
+
+from . import (
+    NeighborListOptions,
+    System,
+)
+
+
+def save(path: Union[str, Path], data: System) -> None:
+    """Save a System object to a file.
+
+    The provided System must contain float64 data and be on the CPU device.
+    Neighbor list requestors will not be saved.
```

Comment:
Why not? You should be able to use the serialization of NL options and automatically save everything.